Fixes for require_keycloak_role

NikosChondros opened the pull request:
First, fix issue with missing padding for the part of the token that contains the json text.
Second, make sure that client is in the loaded dictionary, before attempting to index into it.

NikosChondros committed:
Fixes for require_keycloak_role
First, fix issue with missing padding for the part of the token that contains the json text.
Second, make sure that client is in the loaded dictionary, before attempting to index into it.

pep8speaks commented:
Hello @NikosChondros! Thanks for updating this PR. We checked the lines you've touched for [PEP 8](https://www.python.org/dev/peps/pep-0008) issues, and found:

* In the file [`flask_oidc_ex/__init__.py`](https://github.com/larsw/flask-oidc-ex/blob/cc2873982a9281d25258f061718fd3fe360f79ae/flask_oidc_ex/__init__.py):

> [Line 540:80](https://github.com/larsw/flask-oidc-ex/blob/cc2873982a9281d25258f061718fd3fe360f79ae/flask_oidc_ex/__init__.py#L540): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (104 > 79 characters)
> [Line 541:80](https://github.com/larsw/flask-oidc-ex/blob/cc2873982a9281d25258f061718fd3fe360f79ae/flask_oidc_ex/__init__.py#L541): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (104 > 79 characters)
> [Line 542:80](https://github.com/larsw/flask-oidc-ex/blob/cc2873982a9281d25258f061718fd3fe360f79ae/flask_oidc_ex/__init__.py#L542): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (81 > 79 characters)



##### Comment last updated at 2021-04-08 07:42:16 UTC

NikosChondros committed:
Fixed PEP8 issues

mkonstant committed:
Fixed for require_keycloak_role.
Make sure that 'resource_access' is in the access token, before attempting to search for roles of client.
